### PR TITLE
1722 move models into components

### PR DESCRIPTION
**Pull request by will-byrne**
#### Background

Moving all of the models into `component-model`

#### Any relevant tickets

Partial work of #1722 , we will split each model to their own `component-model-xxxx` after this pr is merged.

### CODE REVIEW

File: config/default.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = {
   pubsweet: {
     components: [
       '@elifesciences/xpub-meca-export',
-      '@elifesciences/xpub-model',
+      '@elifesciences/component-model',
```
Comment (jyenes) [inline config/default.js:13]: not sure if we should rename the xpub-model

File: packages/component-login/client/index.js [hunk 2]
```diff
@@ -67,6 +67,7 @@ class LoginPage extends React.Component {
         LoginPage.setToken(null)
         // TODO expose this error once we have a UI to do so
         this.props.history.push('/login', { error: err.message })
+        // eslint-disable-next-line no-console
```
Comment (jyenes) [inline packages/component-login/client/index.js:70]: why not remove the console.log?
if you need to log anything important, use logger

File: packages/component-login/client/index.test.js [hunk 3]
```diff
@@ -62,6 +62,7 @@ describe('LoginPage component', () => {
 
     await new Promise(resolve => setTimeout(resolve, 0))
     expect(historyMock.push).toHaveBeenCalledWith('/login', { error: 'Nope' })
+    // eslint-disable-next-line no-console
     expect(console.error).toHaveBeenCalled()
```
Comment (jyenes) [inline packages/component-login/client/index.test.js:66]: same here

**Review by jyenes — CHANGES_REQUESTED** [review]

File: packages/component-login/client/index.js (hunk 2 above)
Comment (will-byrne) [reply packages/component-login/client/index.js:70]: will need to change this in a separate PR and add the login test configs.

**Review by jyenes — APPROVED** [review]

**Review by erezmus — APPROVED** [review]

**Pull request merged**